namespace/transitive-dependencies.adoc

```diff
@@ -1,5 +1,981 @@
+
+This list includes all the APIs in Java EE 8,plus a few of the "Java EE" specs that had migrated to Java SE and were remove by link:https://openjdk.java.net/jeps/320[JEP 320],JDK 11+.
+
+```
+jakarta.activation-api-{version}.jar
+jakarta.annotation-api-{version}.jar
+jakarta.batch-api-{version}.jar
+jakarta.ejb-api-{version}.jar
+jakarta.el-api-{version}.jar
+jakarta.enterprise.cdi-api-{version}.jar
+jakarta.enterprise.concurrent-api-{version}.jar
+jakarta.enterprise.deploy-api-{version}.jar
+jakarta.faces-api-{version}.jar
+jakarta.interceptor-api-{version}.jar
+jakarta.jms-api-{version}.jar
+jakarta.json.bind-api-{version}.jar
+jakarta.json-api-{version}.jar
+jakarta.jws-api-{version}.jar
+jakarta.mail-api-{version}.jar
+jakarta.management.j2ee-api-{version}.jar
+jakarta.persistence-api-{version}.jar
+jakarta.resource-api-{version}.jar
+jakarta.security.auth.message-api-{version}.jar
+jakarta.security.enterprise-api-{version}.jar
+jakarta.security.jacc-api-{version}.jar
+jakarta.servlet.jsp.jstl-api-{version}.jar
+jakarta.servlet.jsp-api-{version}.jar
+jakarta.servlet-api-{version}.jar
+jakarta.transaction-api-{version}.jar
+jakarta.validation-api-{version}.jar
+jakarta.websocket-api-{version}.jar
+jakarta.websocket-client-api-{version}.jar
+jakarta.ws.rs-api-{version}.jar
+jakarta.xml.bind-api-{version}.jar
+jakarta.xml.registry-api-{version}.jar
+jakarta.xml.rpc-api-{version}.jar
+jakarta.xml.soap-api-{version}.jar
+jakarta.xml.ws-api-{version}.jar
+javax.inject-{version}.jar
+```
+
+== JAF
+
+|=======
+|Namespace | `javax.activation`
+|EE4J Project | link:https://projects.eclipse.org/proposals/eclipse-project-jaf[Eclipse Project for JAF]
+|Git Repo | link:https://github.com/eclipse-ee4j/jaf[jaf]
+|TCK Git Repo| link:https://github.com/eclipse-ee4j/jaf-tck[jaf-tck]
+|Group Id| jakarta.activation
+|Artifact Id| jakarta.activation-api
+|Latest Spec Version| 1.2
+|=======
+
+=== Dependents
+==== Project
+- link:https://projects.eclipse.org/projects/ee4j.javamail[Eclipse Project for JavaMail]
+- link:https://projects.eclipse.org/proposals/eclipse-implementation-jaxb[Eclipse Implementation of JAXB]
+- link:https://projects.eclipse.org/projects/ee4j.openmq[Eclipse OpenMQ]
+
+==== API
+- mail
+- jax-rs
+- jaxb
+- saaj
+
+
+== Common Annotations
+|=======
+|Namespace | `javax.annotation`
+|EE4J Project | link:https://projects.eclipse.org/projects/ee4j.ca[Eclipse Project for Common Annotations]
+|Git Repo | link:https://github.com/eclipse-ee4j/common-annotations-api[common-annotations-api]
+|Group Id| jakarta.annotation
+|Artifact Id| jakarta.annotation-api
+|Latest Spec Version| 1.3
+|=======
+
+=== Dependents
+==== Project
+- link:https://projects.eclipse.org/projects/ee4j.glassfish[Eclipse Glassfish]
+- link:https://projects.eclipse.org/projects/ee4j.eclipselink[EclipseLink]
+- link:https://projects.eclipse.org/projects/ee4j.jersey[Eclipse Jersey]
+- link:https://projects.eclipse.org/proposals/eclipse-metro[Eclipse Metro]
```

Comment:
why the link points to 'proposal' instead of 'https://projects.eclipse.org/projects/ee4j.metro'? I can find 22 pointers to 'proposal'...